File: ProgramasJava/02-EstructurasdeControl/Ciclos/Menu.java
import javax.swing.JOptionPane;
public class ejercicio1 {
    public static void main(String[] args) {
        String menu ="",opcion ="";
        boolean sentinel = true;
        menu = "===== Menu General =====\n" +
                "1/A)Opcion uno \n" +
                "2/B)Opcion dos\n" +
                "3/C)Opcion tres\n" +
                "4/S)Salir\n" +
                "Elegir Opcion: ";
        do {
            opcion = JOptionPane.showInputDialog(null, menu);
            switch (opcion.toUpperCase()) {
                case "1":
                case "A":
                    JOptionPane.showMessageDialog(null, "Opcion 1");
                    break;
                case "2":
                case "B":
                    JOptionPane.showMessageDialog(null, "Opcion 2");
                    break;
                case "3":
                case "C":
                    JOptionPane.showMessageDialog(null, "Opcion 3");
                    break;
                case "4":
                case "S":
                    JOptionPane.showMessageDialog(null, "Salir");
                    sentinel=false;
                    break;
                default:
                    JOptionPane.showMessageDialog(null, "Opcion no Valida");
            }
        }while(sentinel);
    }
}

File: ProgramasPython/02-EstructurasControl/Ciclos/Menu.py
menu = '''
                ===== Menu General ===== +
                1/A)Promedio de Calificaciones +
                2/B)Promedio de Calificaciones +
                3/C)Promedio de Calificaciones +
                4/S)Salir +
                Elegir Opcion: 
            '''
sentinela = True

while(sentinela):
    opcion = input(menu).upper()

    if opcion == "1" or opcion =='A':
        print('Opcion 1')
    elif opcion == "2" or opcion =='B':
            print('Option 2')
    elif opcion == "3" or opcion =='c':
            print('Option 3')
    elif opcion == "4" or opcion =='S':
            print('Salir')
    sentinela=False 
else:
    print('Opcion no valida') 
print  ('Programa Termiando')